content/web/get-started.md

```diff
@@ -33,14 +33,14 @@ in your npm-based project's web root.
 
 Load the latest release library directly from WorldWind's servers.
 
-    <script src="https://files.worldwind.arc.nasa.gov/artifactory/web {{% latestArtifactoryPath url="https://files.worldwind.arc.nasa.gov/artifactory/api/storage/web"%}}/worldwind.min.js" type="text/javascript"/>
+    <script src="https://files.worldwind.arc.nasa.gov/artifactory/web{{% latestArtifactoryPath url="https://files.worldwind.arc.nasa.gov/artifactory/api/storage/web"%}}/worldwind.min.js" type="text/javascript"/>
 
 ---
 
 ## More Ways to Get Started
 
 - The latest [GitHub release](https://github.com/NASAWorldWind/WebWorldWind/releases/latest) contains all the WorldWind libraries and code examples.
 
-- [Simplest Usage](/tutorials/simplest-example.md) tutorial
```

Comment:
Just for future reference. I had assumed since this is already in the /web folder that I would just add the rest of the path name. But for all links I should be using the full path?

Reply:
I was surprised by this behavior as well. Paths to other pages are of the form /path/from/root, without the .md suffix.